src/vm/eventreporter.cpp

```diff
@@ -721,53 +721,28 @@ void DoReportForUnhandledException(PEXCEPTION_POINTERS pExceptionInfo)
                 // We can also be here when in non-DefaultDomain an exception goes unhandled on a pure managed thread. In such a case,
```

Comment:
Nit, presumably this comment about non-default-domain no longer applies, since there is only one domain now.

Reply:
Thanks for the catch! I'll update the comments accordingly